core/connection_checker.js

```diff
@@ -14,19 +14,18 @@
 goog.module('Blockly.ConnectionChecker');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.Connection');
-goog.require('Blockly.connectionTypes');
+const Connection = goog.require('Blockly.Connection');
+const connectionTypes = goog.require('Blockly.connectionTypes');
+const IConnectionChecker = goog.require('Blockly.IConnectionChecker');
+const registry = goog.require('Blockly.registry');
+const RenderedConnection = goog.requireType('Blockly.RenderedConnection');
```

Comment:
I think these are supposed to be alphabetized with capitals first? I know that's how it was in an old style guide, and that matches the example [here](go/js-style#file-goog-require), but it doesn't say explicitly so up to you.

Reply:
Yeah I was also a bit unclear on this, updated to reflect that convention.